### DISCUSSION TITLE
Uno.UI 4.1.9 is compiled against Microsoft.iOS 15.2.303.55 but only Microsoft.iOS 15.2.302.122 is released

### DISCUSSION THREAD

**Discussion by inforithmics** [unoplatform/uno#8453] [Q&A]
### Current behavior

I tried to use Uno.UI 4.1.9 with net-ios target but following Error Message occours.

Error	CS1705	Assembly 'Uno.UI' with identity 'Uno.UI, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null' uses 'Microsoft.iOS, Version=15.2.303.55, Culture=neutral, PublicKeyToken=84e04ff9cfb79065' which has a higher version than referenced assembly 'Microsoft.iOS' with identity 'Microsoft.iOS, Version=15.2.302.122, Culture=neutral, PublicKeyToken=84e04ff9cfb79065'

Meaning that Uno.UI is compiled against a later preview than 14 maybe release candidate or preview15.

### Expected behavior

It works with a ios workload that is available. like preview.14 which has version 15.2.302.122

### Workaround

Don't know how to force Microsoft.iOS 15.2.302.122

### Affected platforms

iOS

### IDE

Visual Studio 2022

### IDE version

17.1.2

### Relevant plugins

ReSharper

### Anything else we need to know?

tried to target net6.0-ios with ios workload of Preview14.

Progress is visible here. 
https://github.com/Mapsui/Mapsui/pull/1576

**Comment by jeromelaban** [comment]
Thanks for opening the issue. 4.1.9 is built against the upcoming RC1, mostly because it's very difficult to get Preview 14 (see [the known issues](https://github.com/dotnet/maui/releases/tag/6.0.200-preview.14.2)) working properly on CLI. This situation is temporary, until RC1 of .NET 6 Mobile gets released.

At this time, the only way to work around this is to use uno-check, using this PR https://github.com/unoplatform/Uno.XamlBehaviors/pull/24/files as a base.

We hope that RC 1 will make all of this go away.

**Reply by jeromelaban** [reply]
The latest bits for Uno 4.2-dev are built with .NET 6 RC1 bits, which should make this issue go away. It should also not be required to use uno-check to build uno-based applications, only the .NET 6 Mobile standard installation steps will do.
